js/src/main/scala/org.rogach.scallop/ScallopArgListLoader.scala

```diff
@@ -3,5 +3,5 @@ package org.rogach.scallop
 import scala.collection.{Seq => CSeq}
 
 trait ScallopArgListLoader {
-  def loadArgList(args: CSeq[String]): CSeq[String] = args
+  def loadArgList(args: CSeq[String], canReadFromFileOrStdIn: Boolean): CSeq[String] = args
```

Comment:
I'm don't like how `canReadFromFileOrStdIn` is passed through everywhere - it's very crumbesome and touches a lot of code.

Perhaps it would be much easier to add a parameter to Scallop and then use it where needed? For example the way it was done for `appendDefaultToDescription`.